src/server/api/controllers/topics.controller.js

```diff
@@ -26,7 +27,41 @@ const getTopicById = async (id) => {
   }
 };
 
+const editTopic = async (topicId, body) => {
+  return knex('topics')
+    .where({ id: topicId })
+    .update({
+      topic_name: body.topicName,
+      created_at: moment(body.created_at).format(),
+      updated_at: moment(body.updated_at).format(),
```

Comment:
This should be set to _now_. You shouldn't pass this through the body since that would mean you could send the wrong time as a user. Like from the future.